src/product_card-styles/product_card_global.css

```diff
@@ -0,0 +1,27 @@
+:root {
+  --product-card-width: 200px;
+  --product-card-border: 1px;
+  --product-card-padding: 16px;
+  --image-margin-bottom: 40px;
+  --title-margin-bottom: 4px;
+  ---code-margin-bottom: 17px;
+  ---rate-margin-bottom: 24px;
```

Comment:
Replace the dashes '---' with the correct variable syntax '--' for '--rate-margin-bottom'.